### DISCUSSION TITLE
How do you actually use this experiment?

### DISCUSSION THREAD

**Discussion by help137** [zer0Kerbal/PapaKerballinisPizza#40] [Q&A] [answered]
Well, I've had this mod installed for about a week now and I have one big question: How do you actually use this thing?!?! 

I've tried attaching it to various rockets, planes, etc and tried to observe it, there's no option to do so. I've taken it as inventory and tried to deploy it (with pilots, scientist and engineers) all to no avail. Help! I want my pizza! 

-K

**Comment by zer0Kerbal** [comment]
@help137 
thank you for reaching out and thank you for supporting Papa Kerballinis' Pizza Parlour!

Currently it is a glorified `Mystery Goo` experiment - so attach and use like that.

Upcoming release will have a new experiment - (which still should have some better _flavor_ text and _sauce_ 😆 ) but should act the same --- you might need to open the box, but it comes pre-sliced (another bad pun).

I will be updating it this week (right before I start releasing the holiday/seasonal addons) - and will attempt to push out a short (<10m) twitch/youtube video on it, without releasing any spoiled cheese.

**Reply by help137** [reply]
I've tried that, multiple times with it attached to different craft in different spots. I can right click on it and I get the option to Observe the experiment, but when I click on the button, nothing happens. 

-K

**Reply by help137** [reply]
Oh, I'd just like to add,... I'd LOVE to see the video you mentioned was a possibility! 

Thanks,
-K

**Reply by zer0Kerbal** [reply]
> Oh, I'd just like to add,... I'd LOVE to see the video you mentioned was a possibility!
> 
> Thanks, -K

My pleasure @help137 

Recently been streaming short testing clips on twitch and then exporting them to YouTube. I will do that.

I will also taste the pizza today and see if it needs more 🧄 or 🛸 or 🧀

**Comment by zer0Kerbal** [comment, marked as answer]
@help137 got it. pushing out a new release shortly. and a video (combining several mods into the video).

**Reply by zer0Kerbal** [reply]
@help137  - and here is the video. SPOILERS. https://youtu.be/lxn3PSNh_Dg
